apps/rush-lib/assets/rush-init/common/config/rush/version-policies.json

```diff
@@ -86,7 +96,9 @@
      * in some other way, set "exemptFromRushChange" to true to tell "rush change" to ignore the projects
      * belonging to this version policy.
      */
-    "exemptFromRushChange": false
+    "exemptFromRushChange": false,
+
+    /*[LINE "HYPOTHETICAL"]*/ "includeEmailInChangeFile": true
```

Comment:
@octogonz was this line supposed to be `exemptFromRushChange`

Reply:
Is `includeEmailInChangeFile` not supported for `"definitionName": "individualVersion"`?

This was supposed to be the same setting as from `"definitionName": "lockStepVersion"`, just without the `/* */` comment. (We generally don't copy+paste entire paragraphs of `/* */` docs; instead we document the first instance and then omit the comment for later instances).